communis-services/src/main/java/com/rhema/communis/member/FamilyController.java

```diff
@@ -0,0 +1,49 @@
+package com.rhema.communis.member;
+
+import com.rhema.communis.common.CommunisError;
+import com.rhema.communis.common.CommunisResponse;
+import com.rhema.communis.mission.domain.family.Family;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController()
+@RequestMapping("/family")
+public class FamilyController {
```

Comment:
why are we not extending the AbstractController? the REST api formats are not consistent with the framework ones.